### ISSUE TITLE
Would like to share your trained model?

### ISSUE THREAD

**Issue by ShihuaiXu** [jaehanlee-mcl/multi-loss-rebalancing-depth#2]
thank you in advance
besides, when I run the evaluation.py, log shows FileNotFoundError: [Errno 2] No such file or directory: 'D:\\dataset\\depth_map_estimation/NYUv2/test.zip'
would you like to share the test.zip to me?
And dose this code containing training part?

**Comment by jaehanlee-mcl** [comment]
Of course. I will provide everything all you need for training and evaluation. Please wait a little while.

**Comment by ShihuaiXu** [comment]
thank you very much





------------------ Original ------------------
